### PR TITLE
Escape mermaid name & Chore

### PR DESCRIPTION
**Pull request by ethansnow2012**
I was parsing my Next.js project, which have file named [uuid].tsx, which causes mermaid to have error.
Escapes needed.
Because it is too trivial I did not write test for it(should I XD).

And fix some typo in test file.

### CODE REVIEW

File: src/graph.ts [hunk 1]
```diff
@@ -24,6 +24,11 @@ function pathToName(path: string) {
     return name;
 }
 
+function escapeMermaid(str: string) {
+    const escapeChars = '___'
```
Comment (PSeitz) [inline src/graph.ts]: why is this 3 times underscore?

**Comment by PSeitz** [comment]
Can you add a small test for it? As I don't use mermaid and it may break later

File: src/graph.ts (hunk 1 above)
Comment (ethansnow2012) [reply src/graph.ts]: I am not sure what escape chars to use. So I thought 3 is rare enough.
Any suggestion?

**Comment by ethansnow2012** [comment]
Test case added.

File: src/graph.ts (hunk 1 above)
Comment (PSeitz) [reply src/graph.ts]: I doubt there will be conflicts, so readability is more important. So maybe one underscore should be fine

Comment (ethansnow2012) [reply src/graph.ts]: Using one underscore.

**Comment by PSeitz** [comment]
nice, thanks!

**Comment by PSeitz** [comment]
published via `2.1.1`

**Pull request merged**